_toc.yml

```diff
@@ -13,6 +13,7 @@ entries:
       - file: docs/platform/concepts/aiven-node-firewall-configuration
       - file: docs/platform/concepts/authentication-tokens
       - file: docs/platform/concepts/availability-zones
+      - file: docs/platform/concepts/free-plan
```

Comment:
Should the title be 'Free plans'?

Reply:
Or "Free plan services" maybe?

Reply:
Let's leave it as is given we have already put this link in quite a few places :)